test/validate.jl

```diff
@@ -0,0 +1,13 @@
+@test validate("valid.xml", "valid.xsd") == true
+
+@test validate("valid.xml", "valid.xsd") == true
+
+doc = parse_file("valid.xml")
+schema = XMLSchema("valid.xsd")
+
+@test validate("valid.xml", schema) == true
+
+@test validate(doc, schema) == true
+
+@test validate(root(doc), schema) == true
+
```

Comment:
Lets wrap these tests in a testset. This package doesn't make use of testsets currently but I think adding them as we add new tests will make it easier to transition.

Reply:
I will have to look up testsets to do this one. I put in the other changes. Thanks.